modules/RTC/TraceablePeerConnection.js

```diff
@@ -572,6 +569,33 @@ TraceablePeerConnection.prototype._peerMutedChanged = function(
     }
 };
 
+/**
+ * Handles remote source mute, unmute and videoType changed events.
+ *
+ * @param {string} sourceName - The name of the remote source.
+ * @param {boolean} isMuted - The new mute state.
+ * @param {string} videoType - The new videoType value.
+ */
+TraceablePeerConnection.prototype._sourceMutedOrVideoTypeChanged = function(sourceName, isMuted, videoType) {
+    // Ignore presence updates for local sources.
+    const localSource = this.getLocalTracks().find(t => t.getSourceName() === sourceName);
```

Comment:
what's the point in doing local source look up if the method returns anyway when it's true?

Reply:
because we want to ignore presence updates for local sources. Do you want me to rename `localSource` to `isLocalSource` ?

Reply:
why don't break the circuit much earlier at the time when the events are going out of the presence handler? btw. this code ignores them anyway because no remote track will be found